Objects/Converters/ConverterRevit/ConverterRevitShared/Partial Classes/ConvertAnalyticalSurface.cs

```diff
@@ -260,6 +261,172 @@ private Element2D AnalyticalSurfaceToSpeckle(AnalyticalModelSurface revitSurface
 
       return speckleElement2D;
     }
+#else
+    private Element2D AnalyticalSurfaceToSpeckle(AnalyticalPanel revitSurface)
+    {
+
+
+      var speckleElement2D = new Element2D();
+
+      var structuralElement = revitSurface.Document.GetElement(AnalyticalToPhysicalAssociationManager.GetAnalyticalToPhysicalAssociationManager(Doc).GetAssociatedElementId(revitSurface.Id));
```

Comment:
I am not sure if this method returns the actual analytical element, to be checked!